<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptRunConditionEditorDlg</class>
 <widget class="QDialog" name="ScriptRunConditionEditorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="QPushButton" name="btnVarConst">
     <property name="text">
      <string>选择变量</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="editVarVarRight"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="editVarConst"/>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="editValConst"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="editVarVarLeft"/>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="cboVarConst">
     <item>
      <property name="text">
       <string>==</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;=</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnVarLeft">
     <property name="text">
      <string>选择变量</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="cboVarVar">
     <item>
      <property name="text">
       <string>==</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;=</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="btnVarRight">
     <property name="text">
      <string>选择变量</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="radioBtnVarConst">
     <property name="text">
      <string>变量和定值</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="radioBtnVarVar">
     <property name="text">
      <string>变量和变量</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
